src/Svg/Tag/Image.php

```diff
@@ -28,15 +30,15 @@ protected function before($attributes)
 
     public function start($attributes)
     {
-        $document = $this->document;
         $height = $this->document->getHeight();
         $this->y = $height;
 
         if (isset($attributes['x'])) {
-            $this->x = $attributes['x'];
+            $width = $this->document->getWidth();
+            $this->x = Style::convertSize($attributes['x'], $width);
         }
         if (isset($attributes['y'])) {
-            $this->y = $height - $attributes['y'];
+            $this->y = $height - Style::convertSize($attributes['y'], $height);
         }
 
         if (isset($attributes['width'])) {
```

Comment:
You didn't address width/height. The specification doesn't specifically address how this should be interpreted, but based on what I see in the various examples I think these would be OK to calculate relative to the viewport.

Then there's the question of "auto" width/height, or if one value is specified but not another. Those are supposed to be the image's intrinsic value. There's nothing in the library to support that determination now so I think that's something we can address separately.

Reply:
Now added. Have left out complex scenarios for now as suggested.

From my testing, the PDF result did match the image handling of my other SVG renders, but this was the same for pixel sizing also so does not seem to be a percentage-specific issue.